index.rst

```diff
@@ -52,10 +50,12 @@ for details).
 
    download
    projects
-   build-a-brain/index
-   summerschool
+   quickstart
+   Core docs <https://nengo.ai/nengo>
+   GUI docs <https://github.com/nengo/nengo-gui>
```

Comment:
It'd be nice to have the GUI docs at https://nengo.ai/nengo-gui, rather than just the github readme.  Probably beyond the scope of this PR, just jumped out at me as I was reading over this.